### PR TITLE
Make contractions consistent

### PR DESCRIPTION
**Pull request by hannahjump**
For example you are should become 'you're'

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -362,11 +362,11 @@ en:
             href: "https://womensaid.scot/"
           - text: "Get information for children from Childline"
             href: "https://www.childline.org.uk/"
-          - text: "Get information if you are child in Wales"
+          - text: "Get information if you‘re child in Wales"
```
Comment (huwd) [inline config/locales/en.yml:365]: Should that either be:
- Get information if you‘re a child in Wales, or
- Get information if you‘re child is in Wales

Comment (huwd) [inline config/locales/en.yml:365]: Should that either be:
- Get information if you‘re a child in Wales, or
- Get information if your child is in Wales

File: config/locales/en.yml [hunk 2]
```diff
@@ -436,11 +436,11 @@ en:
           - "Yes"
           - "Not sure"
         items:
-          - text: "What to do if you have lost your job"
+          - text: "What to do if you've lost your job"
```
Comment (huwd) [inline config/locales/en.yml:439]:
```suggestion
          - text: "What to do if you‘ve lost your job"
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -436,11 +436,11 @@ en:
...
             href: "https://www.gov.uk/guidance/coronavirus-covid-19-what-to-do-if-you-were-employed-and-have-lost-your-job"
           - text: "What to do if you’re laid-off"
             href: "https://www.gov.uk/lay-offs-short-timeworking"
-          - text: "Find out about your rights if you have been dismissed"
+          - text: "Find out about your rights if you've been dismissed"
```
Comment (huwd) [inline config/locales/en.yml:443]:
```suggestion
          - text: "Find out about your rights if you‘ve been dismissed"
```

File: config/locales/en.yml (hunk 1 above)
Comment (hannahjump) [reply config/locales/en.yml:365]: Good spot 'Get information if you‘re a child in Wales'

File: config/locales/en.yml (hunk 2 above)
Comment (hannahjump) [reply config/locales/en.yml:439]: yep, should be the type of apostrophes we use, wasn't intentional!

File: config/locales/en.yml (hunk 3 above)
Comment (hannahjump) [reply config/locales/en.yml:443]: yep, should be the type of apostrophes we use, wasn't intentional!

File: config/locales/en.yml (hunk 1 above)
Comment (huwd) [reply config/locales/en.yml:365]:
```suggestion
          - text: "Get information if you‘re a child in Wales"
```

**Pull request merged**